### PR TITLE
small style-related changes.

### PR DESCRIPTION
**Pull request by vext01**
See commits.

### CODE REVIEW

File: c_tests/run.rs [hunk 1]
```diff
@@ -150,7 +150,13 @@ fn run_suite(opt: &'static str) {
                 .starts_with("debug_")
     };
     #[cfg(cargo_profile = "debug")]
-    let filter: fn(&Path) -> bool = |p| p.extension().unwrap().to_str().unwrap() == "c";
+    let filter: fn(&Path) -> bool = |p| {
```
Comment (ptersilie) [inline c_tests/run.rs:153]: Does this have anything to do with the long comments is this an unrelated refactor?

Comment (vext01) [reply c_tests/run.rs:153]: It was crashing the unwrap because the `.clang_format` config file has no extension.

Comment (ptersilie) [reply c_tests/run.rs:153]: Fair.

**Comment by vext01** [comment]
Fixed in 3455983. OK to squash?

**Comment by ptersilie** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Comment by vext01** [comment]
@ltratt I forgot about this one. Lukas has pre-reviewed. Just needs merging.

Let's do this before the disambiguation stuff. Then i'll sync that work.
